features/administration.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Bookstore Administration Panel
+  Scenario: Admin Authentication
+    Given I have an admin account
+    When I visit the admin panel url
+    And I enter my admin email
+    And I enter my admin password
+    And I click the "Login" button
+    Then I see the admin panel
+
+  Scenario: Invalid Login
+    Given I do not have an admin account
+    When I visit the admin panel url
+    And I enter my admin email
+    And I enter my admin password
+    And I click the "Login" button
+    Then I see a flash notification that tell me that my email does not exist in the system
+
+  Scenario: Adding a book
+    Given I am logged into the admin panel
+    And I am logged into the site
+    When I visit the admin books url
+    And I click the "New Book" button
+    And I enter the title "Test Book"
+    And I enter the price "135.99"
+    And I select the date "2016-01-26" for "Published date"
+    And I enter the author "Some Person"
+    And I click the "Create Book" button
+    And I visit the admin books url
+    Then I see the book "Test Book"
+    When I visit the public book index
+    Then I see the book "Test Book"
+    And I see the book published date "2016-01-26"
+    And I see the book author "Some Person"
+
+  @javascript
+  Scenario: Deleting a book
+    Given I am logged into the admin panel
+    And I am logged into the site
+    Given there is a book named "Book To Be Deleted"
```

Comment:
given my lack of recent experience with cucumber, i'm not sure if it's a good idea to have two `Given` statements within one `Scenario`, but perhaps this is something you already discussed with Derek (or know to be fine)